kubecost/templates/network-costs/network-costs-daemonset.yaml

```
@@ -76,28 +76,21 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: ELECTION_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
         - name: HOST_PORT
           value: {{ (quote .Values.networkCosts.port) | default (quote 3001) }}
         - name: TRAFFIC_LOGGING_ENABLED
           value: {{ (quote .Values.networkCosts.trafficLogging) | default (quote true) }}
         - name: LOG_LEVEL
           value: {{ .Values.networkCosts.logLevel }}
-        {{- if .Values.networkCosts.softMemoryLimit }}
-        - name: GOMEMLIMIT
-          value: {{ .Values.networkCosts.softMemoryLimit }}
-        {{- end }}
-        {{- if .Values.networkCosts.heapMonitor }}
-        {{- if .Values.networkCosts.heapMonitor.enabled }}
-        - name: HEAP_MONITOR_ENABLED
-          value: "true"
-        - name: HEAP_MONITOR_THRESHOLD
-          value: {{ .Values.networkCosts.heapMonitor.threshold }}
-        {{- if .Values.networkCosts.heapMonitor.outFile }}
-        - name: HEAP_MONITOR_OUTPUT
-          value: {{ .Values.networkCosts.heapMonitor.outFile }}
-        {{- end }}
-        {{- end }}
-        {{- end }}
+        - name: USE_LEADER_KUBERNETES_WATCHERS
+          value: {{ (quote .Values.networkCosts.leaderKubernetesWatcher) | default (quote false) }}
+        - name: USE_HOST_NODE_NETWORK_MAP
+          value: {{ (quote .Values.networkCosts.hostNodeNetworkMap) | default (quote false) }}
```

I've recently been trying to remove any defaults set in templates, and instead always define a default in the `values.yaml`. Which you've already done in this PR.

To me, that feels more explicit and there is no confusion across the `values.yaml` and the template what the default value _should_ be.

```suggestion
        - name: USE_LEADER_KUBERNETES_WATCHERS
          value: {{ (quote .Values.networkCosts.leaderKubernetesWatcher) }}
        - name: USE_HOST_NODE_NETWORK_MAP
          value: {{ (quote .Values.networkCosts.hostNodeNetworkMap) }}
```